### PR TITLE
Added pranjaljain0.md readme file

### PR DESCRIPTION
**Pull request by pranjaljain0**
The markdown file must fulfill these conditions:

- [x] The name of the readme file is your **username.md** if you are making a new creative readme.
- [x] The name of the readme file is the person's username from where it is inspired from.
- [x] The bottom of the readme file contains: `⭐️ From [username](https://github.com/[username])`

### CODE REVIEW

File: dynamic-realtime/pranjaljain0.md [hunk 1]
```diff
@@ -67,5 +67,3 @@
 </a>
 
 ![visitors](https://visitor-badge.glitch.me/badge?page_id=pranjaljain0.pranjaljain0)
-
-⭐️ From [pranjaljain0](https://github.com/pranjaljain0)
```
Comment (DeltaCoderr) [inline dynamic-realtime/pranjaljain0.md]: Well, you need this one
Add it back again.

Comment (pranjaljain0) [reply dynamic-realtime/pranjaljain0.md]: isn't it already added?
damn, I must have confused it,
I'll add again

**Comment by pranjaljain0** [comment]
I've made the recommended change to the file,
Thanks for pointing out.

**Comment by DeltaCoderr** [comment]
👍

**Review by DeltaCoderr — APPROVED** [review]

**Comment by pranjaljain0** [comment]
😘

**Pull request merged**